### DISCUSSION TITLE
🐼 Panda v0.39.2

### DISCUSSION THREAD

**Discussion by anubra266** [chakra-ui/panda#2626] [Announcements]
To upgrade to v0.39.2:

```bash
pnpm up "@pandacss/*"
#or
yarn up '@pandacss/*'
```

### Fixed

- Vue JSX: Fix issue where using custom `jsxFactory` name causes a runtime error
- Fix issue where panda could load unrelated config files that look like a config e.g. `theming-panda.config.ts`
- Fix a cache issue that leads to HMR growing slower in some cases
- Fix an issue where using CSS var in the token references fallback argument would be escaped
- Fix `panda analyze` JSON output serialization

### Added

- Allow nesting (string) token references in the fallback argument

**Comment by peterbartels** [comment]
@anubra266 
After this update I don't get anymore token reference not found warnings like these
`warn [token] Reference not found: spacing.foo in "{spacing.foo}"`

**Reply by astahmer** [reply]
> @anubra266 After this update I don't get anymore token reference not found warnings like these `warn [token] Reference not found: spacing.foo in "{spacing.foo}"`

hey, can you make a minimal repro ? tried here but couldnt reproduce
https://play.panda-css.com/o1OZCPFvvC

**Reply by peterbartels** [reply]
Here in the config recipe, the warning is not showing up when using `p: '{spacing.foo}',`
https://play.panda-css.com/lhbG74qk60

**Reply by astahmer** [reply]
thanks ! will look into that asap 🙏
